pkg/webhook/configuration.go

```diff
@@ -69,10 +70,22 @@ var (
 	five = int32(5)
 	// Ignore means that an error calling the webhook is ignored.
 	Ignore = admissionregistration.Ignore
+	// Fail means that an error calling the webhook causes the admission to fail.
+	Fail = admissionregistration.Fail
 	// transformation function lists to upgrade webhook resources
-	transformSecret = []transformSecretFunc{}
-	transformSvc    = []transformSvcFunc{}
-	transformConfig = []transformConfigFunc{}
+	transformSecret       = []transformSecretFunc{}
+	transformSvc          = []transformSvcFunc{}
+	transformConfig       = []transformConfigFunc{}
+	cvcRuleWithOperations = v1beta1.RuleWithOperations{
+		Operations: []v1beta1.OperationType{
+			v1beta1.Update,
+		},
+		Rule: v1beta1.Rule{
+			APIGroups:   []string{"cstor.openebs.io"},
+			APIVersions: []string{"v1"},
```

Comment:
We have to make similar changes in the Maya webhook no to parse v1 resources.

Reply:
yes